benchmark/benchmain/main.go

```diff
@@ -128,10 +129,11 @@ const (
 	workloadsUnconstrained = "unconstrained"
 	workloadsAll           = "all"
 	// Compression modes.
-	compModeOff  = "off"
-	compModeGzip = "gzip"
-	compModeNop  = "nop"
-	compModeAll  = "all"
+	compModeOff    = "off"
+	compModeGzip   = "gzip"
```

Comment:
Request: add a `compModeGzipV1 = "gzipV1"`, and make `"gzip"` an alias that uses the latest compressor API version we have (i.e. `"gzipV2"` for now).

Reply:
Addressed offline, instead of adding a new flag just replace the behavior of the current flag since it was testing very very old deprecated code.